qutebrowser/config/configdata.py

```diff
@@ -135,7 +135,8 @@ def data(readonly=False):
              "Whether to find text on a page case-insensitively."),
 
             ('startpage',
-             SettingValue(typ.List(), 'https://duckduckgo.com'),
+             SettingValue(typ.List(typ.String(), none_ok=True),
+                          'https://duckduckgo.com'),
```

Comment:
I don't think this should be `none_ok=True`, as it makes no sense to open an empty URL as startpage.